libsolidity/formal/SMTEncoder.cpp

```diff
@@ -3174,10 +3174,24 @@ vector<smtutil::Expression> SMTEncoder::symbolicArguments(FunctionCall const& _f
 		firstParam = 1;
 	}
 
-	solAssert((arguments.size() + firstParam) == functionParams.size(), "");
-	for (unsigned i = 0; i < arguments.size(); ++i)
-		args.push_back(expr(*arguments.at(i), functionParams.at(i + firstParam)->type()));
-
+	if (_funCall.isSuffixCall() && functionParams.size() == 2)
+	{
+		solAssert(!funType->hasBoundFirstArgument());
+		solAssert(arguments.size() == 1);
+		auto const* literalType = dynamic_cast<RationalNumberType const*>(arguments[0]->annotation().type);
+		solAssert(literalType);
+		auto&& [mantissa, exponent] = literalType->fractionalDecomposition();
+		solAssert(mantissa && exponent);
+		solAssert(!mantissa->isNegative() && !exponent->isNegative());
+		args.emplace_back(smtutil::Expression(mantissa->literalValue(nullptr)));
+		args.emplace_back(smtutil::Expression(exponent->literalValue(nullptr)));
```

Comment:
The point of `emplace_back()` was to get rid of the explicit conversion:
```suggestion
		args.emplace_back(mantissa->literalValue(nullptr));
		args.emplace_back(exponent->literalValue(nullptr));
```

Reply:
Ah yes, I mindlessly replaced `push_back()` and forgot about that.